lms/djangoapps/program_enrollments/models.py

```diff
@@ -105,11 +105,6 @@ class Meta(object):
 
         # For each program enrollment, there may be only one
         # waiting program-course enrollment per course key.
-        # This same constraint is implicitly enforced for
```

Comment:
I don't believe this is true anymore, since this is now a `ForeignKey` and not a `OneToOneField`. Please correct me if I am wrong.

Reply:
You are correct here.